feat: allow fire a `FIRE_TWOHAND` gun while driving a unicycle (#6993)
cataclysmbnteam/Cataclysm-BN: docs/en/mod/json/reference/json_flags.md

- `CRAFTRIG` Acts as a dehydrator, vacuum sealer and reloading press for crafting purposes.
  Potentially to include additional tools in the future.
- `CTRL_ELECTRONIC` Controls electrical and electronic systems of the vehicle.

> - `CTRL_WO_HANDS` Allows you fire the gun need two hands while driving. Can only be installed on a part with `STEERABLE` flag.

Reviewer: ```suggestion
- `CONTROL_WITHOUT_HANDS` Allows you to fire two-handed weapons while driving. Can only be installed on a part with `STEERABLE` flag.
```
